src/applications/find-forms/containers/SearchResults.jsx

```diff
@@ -56,8 +73,58 @@ export class SearchResults extends Component {
     focusElement('[data-forms-focus]');
   };
 
+  updateSortedResultsState = () => {
+    const { props, sortTheResults } = this;
+    const clonedResults = cloneDeep(props.results || []);
+
+    const sortedResults = clonedResults.sort((a, b) => sortTheResults(a, b));
```

Comment:
You could use lodash's [orderBy](https://lodash.com/docs/4.17.15#orderBy) if you want! 🙂 Might be a bit more readable since you won't have to return `-1`, `0`, or `1` for that function